apps/condo/domains/common/utils/localesLoader.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path')
+const fs = require('fs')
+const process = require('process')
+const isEmpty = require('lodash/isEmpty')
+const conf = require('@core/config')
+
+let translations = {}
+
+const loadTranslations = () => {
+    const translationsDir = path.join(process.cwd(), 'lang')
```

Comment:
really some of our team workers can use different work dirrectory. it can be a project root or application root.

Reply:
it's better to use __dirname related path

Reply:
@pahaz 
Should I provide it in .env?